### DISCUSSION TITLE
Katas

### DISCUSSION THREAD

**Discussion by LuisAron** [LaunchX-InnovaccionVirtual/CursoIntroPython#163] [Q&A] [answered]
No importa si mando la documentacion junto con las katas resueltas o las tengo que separar, es que yo los junte

**Comment by RodolfoMorquecho** [comment, marked as answer]
La commander acaba de decir en el live que no hay problema porque a final de cuentas todo va a ir en un solo link

**Comment by DanielChavezZ** [comment]
Segun comprendi no hay ningun problema es como te acomodes

**Comment by LuisAron** [comment]
Muchas gracias por su ayuda
